src/lib/directive.js

```diff
@@ -5,7 +5,7 @@ const _ = require('lodash');
 
   angular.module('esn.mailto-handler').directive('opInboxCompose', opInboxCompose);
 
-  function opInboxCompose($parse, $window) {
+  function opInboxCompose($parse, $window, MAILTOHANDLER_CONSTANTS) {
```

Comment:
we need to `require('./constant.js')`

Reply:
I did it in the component, isn't it OK ?